staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter.go

```diff
@@ -36,23 +36,17 @@ import (
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/klog"
 
-	fctypesv1a1 "k8s.io/api/flowcontrol/v1alpha1"
 	fcclientv1a1 "k8s.io/client-go/kubernetes/typed/flowcontrol/v1alpha1"
 )
 
 // Interface defines how the API Priority and Fairness filter interacts with the underlying system.
 type Interface interface {
 	// Handle takes care of queuing and dispatching a request
-	// characterized by the given digest.  The given `noteFn` will be
-	// invoked with the results of request classification.  If Handle
-	// decides that the request should be executed then `execute()`
-	// will be invoked once to execute the request; otherwise
-	// `execute()` will not be invoked.
-	Handle(ctx context.Context,
-		requestDigest RequestDigest,
-		noteFn func(fs *fctypesv1a1.FlowSchema, pl *fctypesv1a1.PriorityLevelConfiguration),
```

Comment:
The side-effects on the Context should be noted.

Reply:
reverted the change